src/Public/js/myradio.form.js

```diff
@@ -356,8 +356,14 @@ window.MyRadioForm = {
         if (typeof tinymce !== 'undefined') {
             tinymce.init(
                 {
-                    selector: "textarea.myury-form-tinymce",
-                    plugins: "anchor autolink charmap code contextmenu fullscreen hr image link lists media tabfocus table wordcount",
+                    selector: "textarea.myradio-form-tinymce",
+                    plugins: "anchor autolink charmap code contextmenu hr image link lists media tabfocus table wordcount",
+                    media_alt_source: false,
+                    menubar: false,
+                    toolbar: "undo redo | styleselect | bold italic underline | alignleft aligncenter alignright alignjustify | bullist numlist outdent indent | link image media charmap | code",
+                    invalid_elements: "script",
```

Comment:
It's probably more a business decision than a technical one, then. There's still no server side validation, and javascript: urls may well still work, but other than that seems vaguely sensible.

I'd also potentially look at the link policy - maybe they should always be target=_blank, or maybe with an exception for stuff on a certain domain?